sourcemash/api/items.py

```diff
@@ -10,6 +10,8 @@
 
 class getVote(fields.Raw):
     def output(self, key, item):
+        if not current_user.is_authenticated():
+            return 0      
```

Comment:
Items need to carry all of the information in item_fields in order to be rendered correctly. The only way to speed item rendering is to eliminate database calls/tries when a user isn't logged in. So, I added that for getVote and it is already present in getUnreadStatus and getSavedStatus.